app/build.gradle

```diff
@@ -0,0 +1,45 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.example.esha.movieapplication"
+        minSdkVersion 15
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        buildConfigField 'String', 'TMDBMOVIEAPIKEY', TMDBMOVIEAPIKEY
+
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+ext{
+
+    okhttp_version = '3.8.0'
+    appcompat_version = '28.0.0-alpha1'
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.android.support:appcompat-v7:28.0.0-alpha1'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
+    implementation "com.android.support:recyclerview-v7:$appcompat_version"
+    implementation 'com.android.support:cardview-v7:28.0.0-alpha1'
```

Comment:
@Esha-Rajbhandari you ahve already defined appcompat_version in ext use this for cardview as well